documentation/modules/creating-migration-plan.adoc

```diff
@@ -24,6 +24,12 @@ You can configure a hook to run an Ansible playbook or custom container image du
 . Specify the following fields:
 
 * *Plan name*: Enter a migration plan name to display in the migration plan list.
++
+[NOTE]
+====
+Plan names cannot include the following character: +
```

Comment:
I think it's not necessary to add a note here, this part is creating plans on UI, there is a clear error message telling users what should be contained in the plan name, see "Plan name can only contain lowercase alphanumeric characters and dashes (-), and must start and end with an alphanumeric character"